### PR TITLE
[CI][SYCL-CTS] List cts_exclude_filter before executing

### PR DESCRIPTION
**Pull request by KornevNikita**
Also allow excluded categories to be builded if CTS_TESTS_TO_BUILD is specified.

### CODE REVIEW

File: .github/workflows/sycl-linux-run-tests.yml [hunk 1]
```diff
@@ -296,14 +296,21 @@ jobs:
...
+      run: |
+        cat $PWD/devops/cts_exclude_filter
     - name: Build SYCL CTS tests
       if: inputs.tests_selector == 'cts'
       env:
         CMAKE_EXTRA_ARGS: ${{ inputs.extra_cmake_args }}
       run: |
+        # Do not exclude any category if passing the CTS_TESTS_TO_BUILD variable.
+        cts_exclude_filter=$([ -n "$CTS_TESTS_TO_BUILD" ] && echo "" || echo "$PWD/devops/cts_exclude_filter")
+
```
Comment (aelovikov-intel) [inline .github/workflows/sycl-linux-run-tests.yml:310]: Can we move that to the "list" step with some renaming there. I think we can just update the env variable in place if we write the updated value to a proper `$GITHUB_SOMETHING`.

Comment (KornevNikita) [reply .github/workflows/sycl-linux-run-tests.yml:310]: Not sure what you mean. Could you provide a bit more info?

Comment (aelovikov-intel) [reply .github/workflows/sycl-linux-run-tests.yml:310]: I think we shouldn't print "excluded" categories if we ultimately won't exclude them.

**Review by aelovikov-intel — APPROVED** [review]

**Comment by KornevNikita** [comment]
Fail: https://github.com/intel/llvm/issues/13636
@intel/llvm-gatekeepers could you please merge

**Pull request merged**